packages/editor/src/plugins/image/index.ts

```diff
@@ -23,6 +23,7 @@ const imageState = object({
   src: upload(''),
   link: optional(object({ href: string('') })),
   alt: optional(string('')),
+  licence: optional(string('')),
```

Comment:
I would probably prefer a id here otherwise we would need a migration to change a license title, right?

Reply:
Without thinking about it too much I think I'd go with an enum with fixed title strings that are the same for all languages for now.

Reply:
Its now saving the key which is usable as ID :+1: